tests/improver-nbhood-vicinity/01-help.bats

```diff
@@ -70,15 +70,17 @@ optional arguments:
                         hours uses a radius of 12000m, etc.
   --ens_factor ENS_FACTOR
                         The factor with which to adjust the neighbourhood size
-                        for more than one ensemble member. If ens_factor = 1.0
-                        this essentially conserves ensemble members if every
-                        grid square is considered to be the equivalent of an
-                        ensemble member.Optional, defaults to 1.0.
+                        for more than one ensemble realization. If ens_factor
+                        = 1.0 this essentially conserves ensemble realizations
+                        if every grid square is considered to be the
+                        equivalent of an ensemble realization.Optional,
```

Comment:
There should probably be a space after `ensemble realization` i.e. `ensemble realization. Optional`